app/assets/stylesheets/_cookie-banner.scss

```diff
@@ -0,0 +1,49 @@
+.cookies-banner {
```

Comment:
What do you think about adding some CSS transition for when it's shown/hidden? Something like:

https://user-images.githubusercontent.com/1292945/172623950-9d85fa74-fb21-4c75-8697-9db26886c7b9.mp4